README-source.md

```diff
@@ -410,6 +410,8 @@ Adding a create method looks very similar.
 Every method you define on a `resource` Zapier converts to the appropriate Trigger, Create, or Search. Our examples
```

Comment:
Thanks for the suggestion @ibolmo ! I've kept it as is, as I don't think it's _that_ important to showcase this, over other nuances. We'll see if people miss it, we'll make it more obvious.